tslint.json

```diff
@@ -31,6 +31,7 @@
                 "jsx-double",
                 "single"
             ]
-        }
+        },
+        "no-implicit-dependencies": false
```

Comment:
Since this rule detects the dependencies in the tests as 'non-dev' dependencies this yields some false positives. Since TSLint is [deprecated](https://blog.palantir.com/tslint-in-2019-1a144c2317a9) and needs to replaced I have opted to disable this for now.

Reply:
Add comment somewhere to do that then.